Configuration/Realm/Tests.xcconfig

```diff
@@ -1,10 +1,11 @@
-SUPPORTED_PLATFORMS = macosx iphonesimulator iphoneos;
+SUPPORTED_PLATFORMS = macosx iphonesimulator iphoneos appletvos appletvsimulator;
 SKIP_INSTALL = YES;
 
 INFOPLIST_FILE = Realm/Tests/RealmTests-Info.plist;
 PRODUCT_BUNDLE_IDENTIFIER = io.Realm.${PRODUCT_NAME:rfc1034identifier};
 
 LD_RUNPATH_SEARCH_PATHS[sdk=iphone*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
+LD_RUNPATH_SEARCH_PATHS[sdk=appletv*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
 LD_RUNPATH_SEARCH_PATHS[sdk=macosx*] = $(inherited) @executable_path/../Frameworks @loader_path/../Frameworks;
```

Comment:
We're not running watchOS tests. When we last looked into it, it wasn't possible to do so, but I don't think we've confirmed that in a while.